src/App.js

```diff
@@ -3,9 +3,32 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Landing from './components/Landing/Landing';
 import Store from './components/Store/Store';
 import Cart from './components/Cart/Cart';
+import GameDetails from './components/GameDetails/GameDetails';
 import './index.css';
 
 const App = () => {
+  const [wishList, setWishList] = React.useState([]);
+  const [showBlackScreen, setShowBlackScreen] = React.useState(false);
+
+  const handleWishList = (clickedGame) => {
+    setWishList((prevWishList) => {
+      if (
```

Comment:
I think you can just do the filter directly, checking if the item exists in the list is kinda useless, I think? 🤔

Reply:
Well the function handles two cases for which different actions are required:
1. game is in the list - in which case, it removes it from the list
2. game is not in the list - in which case, it adds it in the list

I think I need the condition right? Or I am missing something.

Reply:
You are right, I am hallucinating.